CHANGELOG.md

```diff
@@ -14,7 +19,7 @@
 * Adding common `sign_blob()` to service account types and a
   `service_account_email` property. (#421)
 * Improving error message in P12 factory
-  `ServiceAccountCredentials.from_p12_keyfile` when pyOpenSSL is
+  `ServiceAccountCredentials.frogit gadm_p12_keyfile` when pyOpenSSL is
```

Comment:
@jonparrott You fell asleep on your keyboard? Or you just really wanted to frog it

![tgmofa1](https://cloud.githubusercontent.com/assets/520669/15442649/5ef7061a-1e97-11e6-955a-381c8ca91400.gif)